### DISCUSSION TITLE
Ability to duplicate flows

### DISCUSSION THREAD

**Discussion by mighty-hq** [directus/directus#17497] [Archive]
### Summary

Would be awesome to be able to duplicate flows easier. 

### Motivation

Would make things fater

### Detailed Design

Simple "clone" button

### Requirements List

N/A

### Drawbacks

Cant think of any

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
The form is there for a reason. Without the requested detailed information, it's impossible to actually implement a feature request. Moving this to archive..
